datapackage/resource.py

```diff
@@ -279,6 +279,7 @@ def iter(self):
                 parser_class = tabulator.parsers.CSV
             try:
                 table = tabulator.topen(data_path_or_url, with_headers=True,
+                                        encoding=self.metadata.get('encoding'),
```

Comment:
According to http://dataprotocols.org/data-packages/, if no encoding is specified, it should be loaded as UTF-8. I'm not sure if tabulator does this if encoding is undefined, but regardless we should be explicit here IMO. It would be good to add a test as well. The test can be as simple as adding a new fixture in a different encoding and checking if it's loaded correctly. You can see an example in the tests in https://github.com/frictionlessdata/datapackage-py/blob/master/tests/test_resource.py which use `test_helpers.fixture_path()`